WalletWasabi/WabiSabi/Backend/Banning/Prison.cs

```diff
@@ -115,6 +120,16 @@ public bool TryGet(OutPoint utxo, [NotNullWhen(returnValue: true)] out Inmate? i
 			}
 		}
 
+		public bool IsBanned(OutPoint utxo)
+		{
+			if (!TryGet(utxo, out var inmate))
+			{
+				return false;
+			}
+
+			return inmate.Punishment is Punishment.Banned;
```

Comment:
Simply style suggestion. Feel free to ignore.

```suggestion
			return TryGet(utxo, out var inmate) && inmate.Punishment is Punishment.Banned;
```

Reply:
ACK